docs/components.md

```diff
@@ -26,6 +26,7 @@ removed prior to the 1.0 release.
 | [simpleprometheus](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/simpleprometheusreceiver) | [resourcedetection](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor)       |                                                                                                                      |                                                                                                                                  | | |
 | [smartagent](../internal/receiver/smartagentreceiver)                                                                             | [routing](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/routingprocessor)                           |                                                                                                                      |                                                                                                                                  | | |
 | [splunk_hec](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/splunkhecreceiver)              | [span](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanprocessor)                                 |                                                                                                                      |                                                                                                                                  | | |
+| [tcplog](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/tcplogreceiver)                     |                                                                                                                                             |                                                                                                                      |                                                                                                                                  | | |
```

Comment:
currently in alpha too https://github.com/open-telemetry/opentelemetry-collector-contrib/blame/main/receiver/tcplogreceiver/README.md#L8